cosmos/operators/local.py

```diff
@@ -393,24 +384,12 @@ def run_subprocess(self, command: list[str], env: dict[str, str], cwd: str) -> F
 
     def run_dbt_runner(self, command: list[str], env: dict[str, str], cwd: str) -> dbtRunnerResult:
         """Invokes the dbt command programmatically."""
-        try:
-            from dbt.cli.main import dbtRunner
-        except ImportError:
+        if not dbt_runner.is_available():
             raise ImportError(
```

Comment:
Should we change this error type since the ImportError here seems a bit off?

Reply:
Changed in fc38c1536c5b71eb664726c50c7d5619a56b7535